garage.com/src/main/java/ConvertibleCar.java

```diff
@@ -1,6 +1,30 @@
 public class ConvertibleCar extends Car implements ICar {
 
-	public ConvertibleCar(String model, IEngine engine, IIgnition ignitionSystem) {
-		super(model, engine, ignitionSystem);
+	public ConvertibleCar(String model, IEngine engine, IIgnition ignitionSystem, String lowerTop, String raiseTop) {
+		super(model, engine, ignitionSystem, lowerTop, raiseTop);
+	}
+
+	public void start() {
+		
```

Comment:
This is available in the Class, as many of these other methods are.

Reply:
Resolved